public/css/style.css

```diff
@@ -1,117 +1,364 @@
-.section-header {
```

Comment:
The changes break the package page e.g. /pkg/jester - I suppose it's still work in progress.

Reply:
Yeap I 'll have to remove all the old css to test it properly.

Reply:
Please let me know when it's ready :)